ark/utils/spatial_lda_utils.py

```diff
@@ -1,63 +1,91 @@
-import os
-import numpy as np
 import pandas as pd
 
-def check_format_cell_table_args(cell_table, markers, clusters, fovs):
+from ark.settings import BASE_COLS
+from ark.utils.misc_utils import verify_in_list
 
-    """
-    Checks the input arguments of the format_cell_table() function.
+
+def check_format_cell_table_args(cell_table, markers, clusters):
+    """Checks the input arguments of the format_cell_table() function.
 
     Args:
-        cell_table (pd.DataFrame):
-            A pandas DataFrame containing the columns of cell marker frequencies and/or cluster ids.
-        markers (list, str):
+        cell_table (pandas.DataFrame):
+            A DataFrame containing the columns of cell marker frequencies and/or cluster ids.
+        markers (list):
             A list of strings corresponding to marker names.
-        clusters (list, int):
+        clusters (list):
             A list of integers corresponding to cluster ids.
-        fovs (str, int):
-            One of either "all_fovs" indicating all field of views are to be kept, or a list of integers corresponding
-            to the index of each field of view which should be kept.
-
-    Returns:
-        None
     """
 
-    base_cols = [
-        "point",
-        "label",
-        "cell_size",
-        "centroid-0",
-        "centroid-1",
-        "pixelfreq_hclust_cap",
-        "name"
-    ]
-
-    # Check cell table
-    if not isinstance(cell_table, pd.DataFrame):
-        raise ValueError("cell_table must be a pd.DataFrame")
-
-    if not all([True for x in base_cols if x in cell_table.columns]):
-        raise ValueError("cell table must contain the following columns:{}".format(base_cols))
+    # Check cell table columns
+    verify_in_list(cell_table_columns=cell_table.columns.to_list(), required_columns=BASE_COLS)
 
     # Check markers/clusters
-    if all([markers is None, clusters is None]):
+    if markers is None and clusters is None:
         raise ValueError("markers and clusters cannot both be None")
     if markers is not None:
-        if isinstance(markers, list) and len(markers) == 0:
+        if len(markers) == 0:
```

Comment:
What happens if you pass an empty list to verify_in_list? It should still give the right error message, which is the markers that are missing right?

Reply:
Nope, no error is thrown.  
`
import ark.utils.misc_utils as ms
ms.verify_in_list(l1=[], l2=["a", "b", "c"])
`

Reply:
This does though. 
 `ms.verify_in_list(l1=["a", "b", "c"], l2=[])`

Sorry, the function wording is somewhat confusing, is list 1 is supposed to contain the required values?

Reply:
No, you're correct. We check that all the entries in list1 are contained within list2, but we don't check that list1 is non-empty. Can you add a line to that function that checks that both lists are non-empty?